src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/model/ProjectPageModel.kt

```diff
@@ -0,0 +1,153 @@
+import org.wycliffeassociates.otter.common.data.model.Chunk
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.domain.ProjectPageActions
+import org.wycliffeassociates.otter.jvm.app.DefaultPluginPreference
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import org.wycliffeassociates.otter.jvm.app.ui.projectpage.view.ChapterContext
+import org.wycliffeassociates.otter.jvm.app.ui.viewtakes.view.ViewTakesView
+import org.wycliffeassociates.otter.jvm.persistence.WaveFileCreator
+import tornadofx.*
+
+import java.time.LocalDate
+
+class ProjectPageModel {
+    var project: Collection? = null
+
+    // setup model with fx properties
+    var projectTitle: String by property()
+    val projectTitleProperty = getProperty(ProjectPageModel::projectTitle)
+
+    // List of collection children (i.e. the chapters) to display in the list
+    var children: ObservableList<Collection> = FXCollections.observableList(mutableListOf())
+
+    // Selected child
+    var activeChild: Collection by property()
+    val activeChildProperty = getProperty(ProjectPageModel::activeChild)
+
+    // List of chunks to display on the screen
+    var chunks: ObservableList<Chunk> = FXCollections.observableList(mutableListOf())
+
+    var activeChunk: Chunk by property()
+    var activeChunkProperty = getProperty(ProjectPageModel::activeChunk)
+
+    // What record/review/edit context are we in?
+    var context: ChapterContext by property(ChapterContext.RECORD)
+    var contextProperty = getProperty(ProjectPageModel::context)
+
+    // Whether the UI should show the plugin as active
+    var showPluginActive: Boolean by property(false)
+    var showPluginActiveProperty = getProperty(ProjectPageModel::showPluginActive)
+
+    // Keep a view context to start transitions
+    var workspace: Workspace? = null
+
+    val projectPageActions = ProjectPageActions(
+            Injector.directoryProvider,
+            WaveFileCreator(),
+            Injector.collectionRepo,
+            Injector.chunkRepository,
+            Injector.takeRepository
+    )
+
+    init {
+        // TODO: Get from scope
+        Injector
+                .projectRepo
+                .getAllRoot()
+                .observeOnFx()
+                .map {
+                    project = it.first()
+                    project?.let {
+                        // TODO: Use localized resource
+                        projectTitle = it.titleKey
+                    }
+                    project
+                }
+                .observeOn(Schedulers.io())
+                .flatMap {
+                    projectPageActions.getChildren(it)
+                }
+                .observeOnFx()
+                .doOnSuccess {
+                    // Now we have the children of the project collection
+                    children.clear()
+                    children.addAll(it)
+                }
+                .subscribe()
```

Comment:
this needs to be broken up into a function or functions

Reply:
probably a use case

Reply:
@sarabiaj I think ns_new_wizard_usecase has the latest code for this file

Reply:
Yes, will change and get project from scope when integrating with Nathan's code. This is placeholder to get demo data for now